docs/project/library-servicing.md

```diff
@@ -31,7 +31,7 @@ Where the `AssemblyVersion` is set to the old version before updating. To determ
 
 ## Update the package index
 
```

Comment:
Update https://github.com/dotnet/corefx/commit/fe796bbb8f658c98407b189244d37a68d25a6b32#diff-122916076db7087dbc454352fada61eeR107 as well?

Reply:
It should be `https://github.com/dotnet/runtime/commit/d0e4dcc7ebf008e7b6835cafbd03878c3a0e75f8#diff-ec9fd7a62cb0c494d86029014940382cR107`